applications/models.py

```diff
@@ -380,5 +381,9 @@ class ApplicantLetter(TimestampedModel):
     application = models.ForeignKey(BootcampApplication, on_delete=models.CASCADE)
     hash = models.UUIDField(unique=True, default=uuid4)
 
+    def get_absolute_url(self):
```

Comment:
This isn't showing up in django admin

Reply:
It's subtle but it should be on the upper right corner, named "View On Site":

![Screenshot from 2020-06-29 12-04-45](https://user-images.githubusercontent.com/863262/86029123-c40a5800-ba00-11ea-9117-f1b9930b85a4.png)

Reply:
Thanks, I was looking in the wrong spot.  Other than Gavin's latest comment, LGTM